packages/core/src/components/button/_common.scss

```diff
@@ -184,6 +188,10 @@ $non-default-state-selectors: ":last-child, :hover, :active, .pt-active, [class*
     background-image: none;
     color: $dark-button-color-disabled;
   }
+
+  .pt-button-spinner .pt-spinner-head {
+    stroke: $pt-dark-text-color;
```

Comment:
#369 was implemented in the latest release. @greglo please update these colors accordingly.